data/config-schema/src/main/java/net/corda/schema/configuration/StateManagerConfig.java

```diff
@@ -1,15 +1,34 @@
 package net.corda.schema.configuration;
 
 /**
- * Configuration keys to access public parts of the configuration under the {@code corda.stateManager} key.
+ * Configuration keys for State Manager config used to bootstrap a worker.
  */
 public final class StateManagerConfig {
     private StateManagerConfig() {
     }
 
     public static final String STATE_MANAGER = "stateManager";
-
     public static final String TYPE = "type";
+    public static final String STATE_TYPE = "stateType";
+
+    // Keys for state type configuration
+    public enum StateType {
+        FLOW_CHECKPOINT("flowCheckpoint"),
+        P2P_SESSION("p2pSession"),
+        FLOW_MAPPING("flowMapping"),
+        KEY_ROTATION("keyRotation"),
+        TOKEN_POOL_CACHE("tokenPoolCache");
```

Comment:
While we are on it, can we, please, add and enum value for Flow Status?
FAO: @ben-millar and @josephzunigadaly